_includes/head.html

```diff
@@ -5,6 +5,13 @@
 
     <title>Indico - {% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
     <meta name="description" content="{{ site.description }}">
+    <link rel="shortcut icon" type="image/x-icon" href="/img/favicon.ico">
+
+    <meta property="og:url" content="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}">
```

Comment:
Something like `{{ page.title or site.title }}` doesn't work?

Reply:
Yep, this works - I've also used this on the `<title>`